### ISSUE TITLE
Can I translate this series of articles into Chinese?

### ISSUE THREAD

**Issue by yihong0618** [Bogdanp/web-app-from-scratch#4]
Hi~

I want to say thank you, and I really like this series of articles to bring me a lot of help in my study.

I wonder if I could translate your article into Chinese and would declare it reprinted.
Thanks again.

**Comment by Bogdanp** [comment]
Absolutely! Let me know after you've published it and I'll update the posts to link to it.

**Comment by jdzy** [comment]
Well done,hope to renew again!
